libraries/c_sdk/standard/common/include/logging_stack.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Logging Stack V1.0.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file logging_stack.h
+ * @brief Reference implementation of Logging stack as a header-only library.
+ */
+
+#ifndef LOGGING_STACK_H_
+#define LOGGING_STACK_H_
+
+/* Include header for logging level macros. */
+#include "logging_levels.h"
+
+/* FreeRTOS Include. */
+#include "FreeRTOS.h"
+
+/* Standard Include. */
+#include <stdint.h>
+
+/* Metadata information to prepend to every log message. */
+#define LOG_METADATA_FORMAT    "[%s:%d] "
+#define LOG_METADATA_ARGS      __FILE__, __LINE__
```

Comment:
I recently updated the metadata on C-SDK to log the just the file name instead of the absolute file path. The same change can be brought here as well.
Reference: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/demos/logging-stack/logging_stack.h#L39-L45